.travis.yml

```diff
@@ -1,5 +1,6 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.4.0
+  - 2.1.5
+  - 2.4.1
```

Comment:
Oh, it wasn't that important. Just multiple arguments to `Array#concat`. Easy to work around.